File: Hivemind.Ng/templates/entities/PostGameReport.ts
import { InjuryReport } from './InjuryReport';
import { GangLevelUpReport } from './GangLevelUpReport';
import { IncomeReport } from './IncomeReport';

export class PostGameReport {
    public injuries: InjuryReport;
    public experience: GangLevelUpReport;
    public income: IncomeReport;
}

File: Hivemind.Core/Contracts/PostGameReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hivemind.Contracts
{
    public class PostGameReport
    {
        public InjuryReport Injuries { get; set; }
        public GangLevelUpReport Experience { get; set; }
        public IncomeReport Income { get; set; }
    }
}
